pandas/core/frame.py

```diff
@@ -22,7 +20,24 @@
 import numpy as np
 import numpy.ma as ma
 
+import pandas.core.algorithms as algorithms
+import pandas.core.common as com
+import pandas.core.indexes.base as ibase
+import pandas.core.nanops as nanops
+import pandas.core.ops as ops
+import pandas.io.formats.console as console
+import pandas.io.formats.format as fmt
+import pandas.plotting._core as gfx
+from pandas import compat
+from pandas._libs import lib, algos as libalgos
```

Comment:
Internal convention is to group by dependency structure.  This usually means libs first, then util, compat, then core, then ...

Reply:
Actually the pytables module is a pretty good example

Reply:
Cool thanks for the example @jbrockmendel will fix this up!